indexer/cuisines.hpp

```diff
@@ -11,26 +8,17 @@ namespace osm
 {
 using AllCuisines = std::vector<std::pair<std::string, std::string>>;
 
+// This class is threadsafe.
```

Comment:
s/threadsafe/thread-safe/

and we usually write "IS"

Reply:
done